browser/src/map/handler/Map.Keyboard.js

```diff
@@ -741,6 +741,13 @@ L.Map.Keyboard = L.Handler.extend({
 			return true;
 		}
 
+		// CTRL + ALT + O for new document. This needs to be handled by the integrator.
+		if (this._isCtrlKey(e) && e.altKey && e.key.toUpperCase() === 'O') {
+			this._map.fire('postMessage', { msgId: 'OpenDocument' });
```

Comment:
It doesn't use standard convention we have for other commands:    COOL -> Host    UI_xxxx      (but there are already 2 exceptions... so not sure :) )

We have already command to create new document: `UI_CreateFile`
How this is different?
https://sdk.collaboraonline.com/docs/postmessage_api.html#id6

Reply:
Changed the command name to "UI_OpenDocument" in order to follow the rules, thanks :)